pkg/vex/cyclonedx.go

```diff
@@ -66,29 +71,32 @@ func (v *CycloneDX) affected(vuln types.DetectedVulnerability, stmt Statement) b
 			continue
 		}
 		if v.sbom.SerialNumber != link.SerialNumber() || v.sbom.Version != link.Version() {
-			v.logger.Warnw("URN doesn't match with SBOM", zap.String("serial number", link.SerialNumber()),
+			v.logger.Warnw("URN doesn't match with SBOM",
+				zap.String("serial number", link.SerialNumber()),
 				zap.Int("version", link.Version()))
 			continue
 		}
-		if vuln.PkgIdentifier.Match(link.Reference()) && (stmt.Status == StatusNotAffected || stmt.Status == StatusFixed) {
-			v.logger.Infow("Filtered out the detected vulnerability", zap.String("vulnerability-id", vuln.VulnerabilityID),
-				zap.String("status", string(stmt.Status)), zap.String("justification", stmt.Justification))
+		if vuln.PkgIdentifier.Match(link.Reference()) && (stmt.Status == types.FindingStatusNotAffected || stmt.Status == types.FindingStatusFixed) {
+			v.logger.Infow("Filtered out the detected vulnerability",
+				zap.String("vulnerability-id", vuln.VulnerabilityID),
+				zap.String("status", string(stmt.Status)),
+				zap.String("justification", stmt.Justification))
```

Comment:
We have removed information logs for `csaf` and `openvex`. We may also need to remove info logs here.

Reply:
Nice catch. Removed.
https://github.com/aquasecurity/trivy/pull/6084/commits/07b8fcc42b09aa5cc929dec7f54d34f777378c2c